src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -196,7 +196,9 @@ public bool TryGetWellKnownAttribute(WellKnownAttributeFlags flag, out bool valu
             {
                 refKind = isByRef ? RefKind.Ref : RefKind.None;
 
+                type = TupleTypeSymbol.TransformToTupleIfCompatible(type); // temporary shallow unification
```

Comment:
This should be replaced with deep conversion to tuples with assigning names to elements when they are present. Similar to how we deal with Dynamic.

---
In reply to: [62068959](https://github.com/dotnet/roslyn/pull/11047#discussion_r62068959) [](ancestors = 62068959)